src/map/Map.js

```diff
@@ -181,6 +181,11 @@ export var Map = Evented.extend({
 		center = this._limitCenter(toLatLng(center), zoom, this.options.maxBounds);
 		options = options || {};
 
+		if (options.zoom && options.zoom.animate && options.zoom.duration) {

```

Comment:
This is not how options passed to `setView` work. Just use `options.duration` instead. Am I missing a documented use case here?